src/main/resources/swagger.api/mosaic.yaml

```diff
@@ -6,6 +6,16 @@ info:
 servers:
   - url: /mosaic
 paths:
+  ### Validation ###
+  /validate:
+    description: Validation API
```

Comment:
Please add more details in order to explain why separate new enpoint is added instead of spring actuator and how it is used to share knowledge that we have already about it